py/desiutil/depend.py

```diff
@@ -126,6 +126,48 @@ def iterdep(header):
 
     raise StopIteration
 
+def add_dependencies(header, module_names=None):
+    '''Adds DEPNAMnn, DEPVERnn keywords to header for imported modules
+    
+    Args:
+        header : dict-like object, e.g. astropy.io.fits.Header
+    
+    Options:
+        module_names : list of module names to check
+            if None, check a list of known potential DESI dependencies
+        
+    Only adds the dependency keywords if the module has already been
+    previously loaded in this python session.  Uses module.__version__
+    if available, otherwise "unknown (/path/to/module/)".
+    '''
+    import sys
+    import importlib
+    
+    setdep(header, 'python', sys.version.replace('\n', ' '))
+    
+    if module_names is None:
+        module_names = [
```

Comment:
I don't think we want to "bury" this list inside this function.  It should be a module-level variable or read from a configuration file.